registry.json

```diff
@@ -1,18 +1,302 @@
 [
-      {
-       "ticker": "ALTZ",                                                       
-       "poolId": "46ebbc0d27d9169cae12d12dcf8f6cacbcf27937c67ae4350abbcaba",  
+  {
+      "ticker": "SWIFT",                                                      
```

Comment:
There are a lot more changes than just removing your pool. Please can you change to just removing the pool you intend to?